tests/plugins/link/link.js

```diff
@@ -592,7 +592,53 @@
 			correctInput: '123456789',
 			incorrectLinkAssertionCallback: assertIncorrectLinks,
 			correctLinkAssertionCallback: assertCorrectLinks
-		} )
+		} ),
+
+		// (#2478)
+		'test Ctrl+K keystroke': function() {
+			var bot = this.editorBots.noValidation,
+				editor = bot.editor;
+
+			bot.setData( '', function() {
+				editor.once( 'dialogShow', function( evt ) {
+					resume( function() {
+						var dialog = evt.data;
+
+						dialog.hide();
+						assert.areSame( 'link', dialog._.name );
+					} );
+				} );
+
+				editor.editable().fire( 'keydown', new CKEDITOR.dom.event( {
+					keyCode: 75,
+					ctrlKey: true
+				} ) );
+				wait();
+			} );
+		},
+
+		// (#2478)
+		'test Ctrl+L keystroke': function() {
+			var bot = this.editorBots.noValidation,
+				editor = bot.editor;
+
+			bot.setData( '', function() {
+				editor.once( 'dialogShow', function( evt ) {
+					resume( function() {
+						var dialog = evt.data;
+
+						dialog.hide();
```

Comment:
Instead of hiding dialog here, you could move the whole teardown login into `tearDown` test function. Something like:

```javascript
  var dialog = CKEDITOR.dialog.getCurrent();
  if ( dialog ) {
    dialog.hide();
  }
```
It also applies to the second unit test.

Reply:
I'm not sure if it's necessary. Every other test in this file hides dialog on its own. Having two tests that are using `tearDown` for it would break consistency.

Reply:
Not if you refactor other tests. However, following the rule "If it ain't broke, don't fix it", lets deal with important stuff, not test refactoring.